.eslintrc

```diff
@@ -27,6 +27,13 @@
     "react/jsx-uses-vars": [1],      // avoid no-unused-vars for imports used as react components
     "react/jsx-uses-react": [1],     // avoid no-unused-vars for importing React
     "react/react-in-jsx-scope": [1], // make sure React is in scope when writing jsx
-    "no-var": ["error"]
+    "no-var": ["error"],
+    "jsx-a11y/label-has-for": [ 2, {
```

Comment:
We should use "error" instead of "2" (https://eslint.org/docs/user-guide/configuring) 
I see we have used "1" instead of "warn" above. This should be changed, but not necessarily in this PR.